console/src/components/Common/utils/v1QueryUtils.js

```diff
@@ -1,8 +1,9 @@
 export const getRunSqlQuery = (sql, shouldCascade, readOnly) => {
+  const sqlWithSemicolon = sql[sql.length - 1] !== ';' ? sql + ';' : sql;
```

Comment:
We can move this to sqlutils as a function `terminateSql`. You also need to trim the sql first.